src/main/webapp/i18n/en/assessmentInstructions.json

```diff
@@ -12,7 +12,7 @@
             }
         },
         "problemStatement": "Problem Statement",
-        "sampleSolution": "Example Solution",
+        "sampleSolution": "Sample Solution",
```

Comment:
We call it `Example Solution` on purpose to emphasize that it is one exemplary solution, but not the only solution.

`Sample solution` translates to `Musterlösung` in German, which we would like to avoid, because some users might associate this term with other solutions automatically being wrong

Reply:
Ah, understood! I will revert these changes then.